src/modules/init/index.ts

```diff
@@ -16,14 +16,20 @@ const { mapSeries } = Bluebird
 
 const templates = {
   'store-theme': 'store-theme',
+  'render-guide': 'render-guide',
+  'masterdata-graphql-guide': 'masterdata-graphql-guide',
 }
 
 const titles = {
   'store-theme': 'Store Theme',
+  'render-guide': 'Render Guide',
```

Comment:
What is the use case this app solves?

I don't think it's desirable to have an init with no clear purpose.

Reply:
It includes many examples on how to use the "render" framework.